ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿//29вариант
+
+using System;
+public enum daysOfTheWeek
+{
+    Monday    = 1,
+    Tuesday   = 2,
+    Wednesday = 3,
+    Thursday  = 4,
+    Friday    = 5,
+    Saturday  = 6,
+    Sunday    = 7
+}
+namespace ConsoleApp1
+{
+    //правочка
+    class Program
+    {
+        static void functionNumberOfDays() 
+        {
+            Console.WriteLine("please enter the day ");
+
+            var userInput = Console.ReadLine();
+
+            //проверка введёных данных 
+            //start            
+            Int32 val;
+            if (!Int32.TryParse(userInput, out val))
+            {
+                Console.WriteLine("not a number entered");
+                return;
+            }
```

Comment:
Предлагаю заменить это на обработчик исключений

```suggestion
            try {
                val = int.Parse(userInput);
            }
            except {
                Console.WriteLine("not a number entered");
                return;
            }
```

Reply:
ой а что такое обработчик исключений?

Reply:
Это из будущих лекций, просто закоммить изменения